src/Npgsql/TypeHandling/NpgsqlTypeHandler.cs

```diff
@@ -70,6 +75,11 @@ public abstract class NpgsqlTypeHandler
         /// <returns>The fully-read value.</returns>
         internal abstract TAny Read<TAny>(NpgsqlReadBuffer buf, int len, FieldDescription fieldDescription = null);
 
+        internal TAny ReadEntry<TAny>(NpgsqlReadBuffer buf, int len, FieldDescription fieldDescription = null)
```

Comment:
The new `Entry` suffix isn't very clear naming-wise - what it really seems to mean is "read this thing which may be a nullable". Maybe just `ReadNullable` or `ReadWrapper`?

Reply:
I know, but the best thing came to my mind was an entry point. Will do.